colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -41,13 +41,20 @@
 ZERO_AXIS, DP_AXIS, TP_AXIS = 0, 1, 2
 
 
-def get_param_info(optim: Optimizer):
+def get_param_info(model: nn.Module, optim: Optimizer):
     # Get a backup of necessary information of parameters for future use, which includes:
     # 1. A mapping from integer param_id to param32 shape.
 
+    param_info = {"id2shape": {}, "name2shape": {}}
+    for m_name, m_var in model.named_modules():
```

Comment:
Why not use `model.named_parameteres()` directly?

Reply:
updated.